src/store/reducers/index.js

```diff
@@ -142,6 +142,7 @@ const reducer = (state = initialState, action) => {
         // ...state,
         // videos: [...state.videos, action.payload],
         postVideoStart: false,
+        singleVideo: action.payload,
```

Comment:
I attempted running the payload through singleVideo earlier and it didn't work (due to missing the typo on video).